### PR TITLE
Add Usage Example to tf.math.log_sigmoid

### PR DESCRIPTION
**Pull request by malder08**
Added a usage example to tf.math.log_sigmoid

### CODE REVIEW

**Comment by gbaned** [comment]
@malder08 thank you, it is failing doctest can you please check here for [logs](https://source.cloud.google.com/results/invocations/3012f368-334d-4b14-89f1-6c96e19860ab/targets/%2F%2Ftensorflow%2Ftools%2Fdocs:tf_doctest/tests).

Please run the doctest locally as mentioned here in the [contributor guidelines](https://www.tensorflow.org/community/contribute/docs_ref).

File: tensorflow/python/ops/math_ops.py [hunk 1]
```diff
@@ -3305,6 +3305,15 @@ def log_sigmoid(x, name=None):
   Specifically, `y = log(1 / (1 + exp(-x)))`.  For numerical stability,
   we use `y = -tf.nn.softplus(-x)`.
 
+  Usage Example:
+   >>>import tensorflow as tf
```
Comment (RichardXiao13) [inline tensorflow/python/ops/math_ops.py]: You can remove this import statement as it already exists in the file.

File: tensorflow/python/ops/math_ops.py [hunk 2]
```diff
@@ -3305,6 +3305,15 @@ def log_sigmoid(x, name=None):
...
+   >>>tensor1 = [0.0, 1.0, 2.0, 3.0, 4.0, 5.0]
+   >>>tensor2 = [2.0, 5.7]
+   >>>print(tf.math.log_sigmoid(tensor1, name='log_sigmoid_tensor1'))
+   Tensor("log_sigmoid_tensor1_5:0", shape=(6,), dtype=float32)
```
Comment (RichardXiao13) [inline tensorflow/python/ops/math_ops.py]: Make sure you run this in TF 2.0 as this is the output for TF 1.0. Then, you can make changes accordingly.

File: tensorflow/python/ops/math_ops.py [hunk 3]
```diff
@@ -3305,6 +3305,15 @@ def log_sigmoid(x, name=None):
...
+   >>>print(tf.math.log_sigmoid(tensor2, name='log_sigmoid_tensor2'))
+   Tensor("log_sigmoid_tensor2_1:0", shape=(2,), dtype=float32)
```
Comment (RichardXiao13) [inline tensorflow/python/ops/math_ops.py]: Same here. Just provide the output for TF 2.0.

**Review by RichardXiao13 — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/math_ops.py (hunk 1 above)
Comment (malder08) [reply tensorflow/python/ops/math_ops.py]: Sorry about that, the new commit should solve these issues

File: tensorflow/python/ops/math_ops.py [hunk 4]
```diff
@@ -3305,6 +3305,14 @@ def log_sigmoid(x, name=None):
   Specifically, `y = log(1 / (1 + exp(-x)))`.  For numerical stability,
   we use `y = -tf.nn.softplus(-x)`.
 
+  Usage Example:
+   >>> tensor1 = [0.0, 1.0, 2.0, 3.0, 4.0, 5.0]
+   >>> tensor2 = [2.0, 5.7]
+   >>> print(tf.math.log_sigmoid(tensor1, name='log_sigmoid_tensor1'))
```
Comment (boronhub) [inline tensorflow/python/ops/math_ops.py]: You don't need to use print() here.

File: tensorflow/python/ops/math_ops.py [hunk 5]
```diff
@@ -3305,6 +3305,14 @@ def log_sigmoid(x, name=None):
...
+   tf.Tensor([-0.6931472  -0.31326166 -0.12692805 -0.04858733 -0.01814996 -0.00671535], shape=(6,), dtype=float32)
```
Comment (boronhub) [inline tensorflow/python/ops/math_ops.py]: Please correct the output format, as seen [here](https://www.tensorflow.org/community/contribute/docs_ref). Same for the next output.

**Review by boronhub — CHANGES_REQUESTED** [review]
Hi ! I have left a few comments, kindly go over them once.

File: tensorflow/python/ops/math_ops.py (hunk 4 above)
Comment (malder08) [reply tensorflow/python/ops/math_ops.py]: Thanks for the help, my most recent commit should fix these issues.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
No need to have 2 different similar examples (`tensor1` and `tensor2` are only different in the values they contain). Please use only one.

Also, it is better to use significant values in the tensor, for example, `[-100, 0, 100]`, as those would show the behavior of the function at important cornercases

**Comment by malder08** [comment]
Ok that makes sense, thanks. My most recent commit removed the second example and changed the values in the tensor to be more significant.

**Review by mihaimaruseac — APPROVED** [review]

**Comment by ravikyram** [comment]
@malder08 

Can you please check build failures? Thanks!

**Comment by gbaned** [comment]
@malder08 gentle ping

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Comment by malder08** [comment]
So sorry for the late response, I seem to have missed the first ping. I've been having some trouble getting the lab test to work on my computer so I can try to see the error to fix. When I am able to get that working I'll reopen the pull request. Again, sorry and thanks!
